www_src/pages/page/page.jsx

```diff
@@ -164,6 +166,10 @@ var Page = React.createClass({
           state.currentElementId = id;
         }
         this.setState(state);
+
+        setTimeout(function() {
```

Comment:
Gotcha. In that case, I'd recommend we match it to the transition time in the CSS (`0.2s`) and add a comment that explains why JIK that animation value ever gets changed.